src/styles/LoginPage.module.scss

```diff
@@ -1,7 +1,12 @@
 @import "colors";
 
+$btn-padding-y: 0.5rem;
```

Comment:
what if I need padding of 0.5rem on the x axis? give some granularity to the padding var names. like xs,s,m,l,xl

Reply:
lets move all sass variables to separate folder and have generic names. 
so this would be like `padding-xs`